orchagent/main.cpp

```diff
@@ -134,6 +139,18 @@ void initSaiApi()
     sai_log_set(SAI_API_ACL,                    SAI_LOG_NOTICE);
 }
 
+string getTimestamp()
```

Comment:
later we can move this to common, im using similar stuff in sairedis for recording